app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -277,6 +280,17 @@ public void onClick(View v) {
     }
 
     @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (requestCode == 1 && resultCode == RESULT_OK) {
```

Comment:
It should be a variable an not a 'hardcoded' value in both the main activity and here